types/node/globals.d.ts

```diff
@@ -601,24 +601,24 @@ declare namespace NodeJS {
 
     interface WritableStream extends EventEmitter {
         writable: boolean;
-        write(buffer: Buffer | string, cb?: Function): boolean;
-        write(str: string, encoding?: string, cb?: Function): boolean;
-        end(cb?: Function): void;
-        end(buffer: Buffer, cb?: Function): void;
-        end(str: string, cb?: Function): void;
-        end(str: string, encoding?: string, cb?: Function): void;
+        write(buffer: Buffer | string, cb?: (err?: Error | null) => void): boolean;
```

Comment:
`write` and `end` allow also `Uint8Array`

Reply:
hnnng